src/marketplace-custom-config.js

```diff
@@ -1,77 +1,150 @@
 /*
  * Marketplace specific configuration.
+ *
+ * Every filter needs to have following keys:
+ * - **id**: unique id of the filter
+ * - **label**: The default label of the filter
+ * - **type**: string that represents one of the existing filter components
+ * - **group**: is this 'primary' or 'secondary' filter?
+ * - **queryParamNames**: describes parameters to be used with API (e.g. 'price' or 'pub_amenities')
```

Comment:
Is this param name used for both the FTW URL and in the Marketplace API call? Is it possible to have a localised or otherwise different param name in FTW, but keep the param name for the API?

I don't know if having them different is a useful thing to support, and people can always hack the actual param names in the duck file, but at least mentioning here that this config changes both the FTW URL and the API params would be good.

Reply:
Those can be different - although, we mostly seem to follow API.
Currently, we have one exception to the rule that param names are the same in both: 'dates'. (react-dates internal output for date range is `{ dates: { startDate, endDate } }`, which is probably the reason it has been used with that filter.

I'll change the comment to this
```
 * - queryParamNames: Describes parameters to be used with queries
 *                    (e.g. 'price' or 'pub_amenities'). Most of these are
 *                    the same between webapp URLs and API query params.
 *                    You can't change 'dates', 'price', or 'keywords'
 *                    since those filters are fixed to a specific attribute.
 ```